### PR TITLE
Removes snowflake code

### PR DESCRIPTION
**Pull request by Probe1**
It was unchecked, I don't see it being called anywhere, and it's cancerous.

Double checked, don't see the icon = path in the files.  So even that's already gone.

### CODE REVIEW

**Review by PJB3005 — APPROVED** [review]

**Comment by 9600bauds** [comment]
Oh god I want to see the sprites for these

**Comment by Probe1** [comment]
@PsiOmegaDelta do you have any skeletons in your sprite closet?

**Comment by Exxion** [comment]
Or just go look at customitems.dmi before Comic removed it like a year ago

**Comment by 9600bauds** [comment]
oh boy

dreammaker_2016-12-23_12-59-28

Most of these are bad, but some are... salvageable? Maybe?

https://a.uguu.se/wpNASUY5kObe_custom_items.dmi
https://a.uguu.se/ssLV41hRB7Mr_custom_items-1.dmi

**Comment by Probe1** [comment]
sslv41hrb7mr_custom_items-1

wpnasuy5kobe_custom_items

**Comment by ultraplen** [comment]
Do they players these "belong" to spawn with the item in their bag every round or something ?

**Comment by Probe1** [comment]
They had some method that has been lost in time like tears
in
the
rain

**Comment by Exxion** [comment]
In theory, yes, but we don't have any players set up to get any

I think we should use the system to punish chronic shitters with unremovable garbage

**Comment by Probe1** [comment]
👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👍 

Trash bans

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -1,753 +0,0 @@
...
-	icon_state = "fountainpen"
-
-/obj/item/fluff/victor_kaminsky_1 //chinsky: Victor Kaminski
-	name = "golden detective's badge"
-	desc = "Nanotrasen Security Department detective's badge, made from gold. Badge number is 564."
-	icon_state = "victor_kaminsky_1"
-
-/obj/item/fluff/victor_kaminsky_1/attack_self(mob/user as mob)
-	for(var/mob/O in viewers(user, null))
-		O.show_message("[user] shows you: [bicon(src)] [name].", 1)
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:107]: >no cool down calling viewers()

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -1,753 +0,0 @@
...
-	name = "Graduation Lighter"
-	desc = "A silver engraved lighter with 41 on one side and Tharsis University on the other. The lid reads Fay Sullivan, Cybernetic Engineering, 2541"
-	icon = 'icons/obj/custom_items.dmi'
-	icon_state = "gradzippo"
-	icon_on = "gradzippoon"
-	icon_off = "gradzippo"
-
-/obj/item/weapon/lighter/zippo/fluff/executivekill_1 //executivekill: Hunter Duke
-	name = "Gonzo Fist zippo"
-	desc = "A Zippo lighter with the iconic Gonzo Fist on a matte black finish."
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:164]: a dishonor to the name

File: code/modules/customitems/item_defines.dm [hunk 3]
```diff
@@ -1,753 +0,0 @@
...
-	desc = "A pristine purple comb made from flexible plastic. It has a small K etched into its side."
-	w_class = W_CLASS_TINY
-	icon = 'icons/obj/custom_items.dmi'
-	icon_state = "purplecomb"
-	item_state = "purplecomb"
-
-	attack_self(mob/user)
-		if(user.r_hand == src || user.l_hand == src)
-			for(var/mob/O in viewers(user, null))
-				O.show_message(text("<span class='warning'>[] uses [] to comb their hair with incredible style and sophistication. What a guy.</span>", user, src), 1)
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:189]: Nice

File: code/modules/customitems/item_defines.dm [hunk 4]
```diff
@@ -1,753 +0,0 @@
-hi
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:1]: Hi

File: code/modules/customitems/item_defines.dm [hunk 5]
```diff
@@ -1,753 +0,0 @@
...
-	icon_state = "lithiumflask"
-	volume = 50
-
-/obj/item/weapon/reagent_containers/glass/beaker/large/fluff/nashida_bishara_1 //rukral:Nashida Bisha'ra
-	name = "Nashida's Etched Beaker"
-	desc = "The message: 'Please do not be removing this beaker from the chemistry lab. If lost, return to Nashida Bisha'ra' can be seen etched into the side of this 100 unit beaker."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "beakerlarge"
-	g_amt = 5000
-	volume = 100
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:242]: >start with extra large beaker

File: code/modules/customitems/item_defines.dm [hunk 6]
```diff
@@ -1,753 +0,0 @@
...
-	reagents.add_reagent(OXYCODONE, 15)
-	update_icon()
-	return
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/attack_self(mob/user as mob)
-	to_chat(user, "<span class='notice'>You click \the [src] but get no reaction. Must be dead.</span>")
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/attack(mob/M as mob, mob/user as mob)
-	if (user.ckey != "nerezza") //Because this can end up in the wrong hands, let's make it useless for them!
-		to_chat(user, "<span class='notice'>You click \the [src] but get no reaction. Must be dead.</span>")
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:300]: Special snowflake hypospray only usable by my OC DONUT STEEL

File: code/modules/customitems/item_defines.dm [hunk 7]
```diff
@@ -1,753 +0,0 @@
...
-		return
-	if (!( istype(M, /mob) ))
-		return
-	if (reagents.total_volume)
-		if (M == user && user.ckey == "nerezza") //Make sure this is being used by the right person, for the right reason (self injection)
-			visible_message("<span class='notice'>[user] presses their </span>\
-				penlight against their skin, quickly clicking the button once.", \
-				"<span class='notice'>You press the disguised autoinjector against your skin and click the button. There's a sharp pain at the injection site that rapidly fades.</span>", \
-				"You hear a rustle as someone moves nearby, then a sharp click.")
-		if (M != user && user.ckey == "nerezza") //Woah now, you better be careful partner
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:313]: So I can use it on myself if I'm not them what???

**Comment by kol1th** [comment]
Flaming welding helmet WHEN

File: code/modules/customitems/item_defines.dm [hunk 8]
```diff
@@ -1,753 +0,0 @@
...
-	item_state = "leatherjack"
-	color = "leatherjack"
-
-/obj/item/clothing/suit/storage/labcoat/fluff/burnt //Jamini: Edwin Atweeke
-	name = "burnt labcoat"
-	desc = "This lab coat has clearly seen better, less burnt, days."
-	icon = 'icons/obj/custom_items.dmi'
-	icon_state = "labcoat_burnt_open"
-
-/obj/item/clothing/suit/armor/vest/fluff/deus_blueshield //deusdactyl
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:518]: >start with free armor

File: code/modules/customitems/item_defines.dm [hunk 9]
```diff
@@ -1,753 +0,0 @@
...
-	name = "emerald necklace"
-	desc = "A brass necklace with a green emerald placed at the end. It has a small inscription on the top of the chain, saying \'Foster\'"
-	icon = 'icons/obj/custom_items.dmi'
-	icon_state = "ty_foster"
-	flags = FPRINT|TABLEPASS
-	w_class = W_CLASS_TINY
-
-//////////// Shoes ////////////
-
-/obj/item/clothing/shoes/magboots/fluff/susan_harris_1 //sniperyeti: Susan Harris
```
Comment (Iamgoofball) [inline code/modules/customitems/item_defines.dm:685]: >free magboots

**Review by Iamgoofball — APPROVED** [review]
10/10 not OP at all

**Comment by ultraplen** [comment]
@kol1th just add an attacked_by on the welding mask with an orange crayon or something to change the skin to the flame one

File: code/modules/customitems/item_defines.dm (hunk 6 above)
Comment (Probe1) [reply code/modules/customitems/item_defines.dm:300]: This was the one that I remembered this morning and made me track this file down.

**Comment by Probe1** [comment]
That's a pretty good idea.

File: code/modules/customitems/item_defines.dm (hunk 7 above)
Comment (Exxion) [reply code/modules/customitems/item_defines.dm:313]: No. You can only use it if you are them, and only on yourself.

File: code/modules/customitems/item_defines.dm [hunk 10]
```diff
@@ -1,753 +0,0 @@
...
-		for(var/datum/reagent/R in reagents.reagent_list)
-			to_chat(usr, "<span class='notice'>You examine the penlight closely and see that it has [R.volume] units of [R.name] stored.</span>")
-	else
-		to_chat(usr, "<span class='notice'>You examine the penlight closely and see that it is currently empty.</span>")
-
-//End strange penlight
-
-/obj/item/weapon/card/id/fluff/asher_spock_2 //Nerezza: Asher Spock
-	name = "Odysses Specialist ID card"
-	desc = "A special identification card with a red cross signifying an emergency physician has specialised in Odysseus operations and maintenance.\nIt grants the owner recharge bay access."
```
Comment (Exxion) [inline code/modules/customitems/item_defines.dm:336]: 1. It claims to do something stupid
2. It doesn't actually do it
3. There's a fucking typo in the name

File: code/modules/customitems/item_defines.dm [hunk 11]
```diff
@@ -1,753 +0,0 @@
...
-	desc = "A meticulously clean police uniform belonging to Precinct 31, Outer Light Colony. The word \"RETIRED\" is engraved tastefully and professionally in the badge below the number, 501."
-	icon = 'icons/obj/custom_items.dmi'
-	icon_state = "retpoluniform"
-	color = "retpoluniform"
-
-//////////// Weapons ////////////
-
-///// Colt Peacemaker - Ana Ka'Rimah - SueTheCake
-
-//obj/item/weapon/gun/energy/stunrevolver/fluff/ana_peacemaker
```
Comment (Exxion) [inline code/modules/customitems/item_defines.dm:742]: Alright this was commented out but come the fuck on

**Comment by Intigracy** [comment]
0fb39728-c903-11e6-8a5f-3d11d3a65099

Only ones worth keeping tbh.

Going clockwise starting at the ID:

Trader ID
Crayons on welding helmets = flaming homo (or give it to mechanics)
These flashlights are nice
Convert the books we currently have to use this more sleek sprite?

**Comment by ultraplen** [comment]
The ivory handle revolvo is pretty cool aswell tbh

**Comment by Intigracy** [comment]
Could add it as an alternative detective revolver, true.

**Comment by ultraplen** [comment]
Maybe the bone/ivory revolver skin could be added to the revolver by grabbing a skulle/bone/remains from a monkey then crafting the grips then attacked by. Minor item interactions are what makes this game so magical tbh.

**Comment by PsiOmegaDelta** [comment]
@Probe1 Since I don't sprite that's fairly unlikely.

**Comment by Probe1** [comment]
😆

**Comment by ComicIronic** [comment]
I actually stole one of the custom sprites for the atmos tech custom magboots. The rest deserved what they got.

**Comment by Probe1** [comment]
Yea I caught that.  That's why the atmos magboots have that weird outline.  I always wondered.

**Pull request merged**